storybook/storyLoader.js

```diff
@@ -0,0 +1,31 @@
+
+// template for doT (https://github.com/olado/doT)
```

Comment:
~~I don’t really understand this part.~~

Ohhhh, I see. This is autogenerated 👍